recipes/full_finetune_distributed.py

```diff
@@ -290,10 +311,12 @@ def _setup_model(
         # Ensure no params and buffers are on meta device
         utils.validate_no_params_on_meta_device(model)
 
-        if enable_activation_checkpointing:
+        # original activation checkpointing (full) - flip the condition above
+        if enable_activation_checkpointing and ac_mode is None:
```

Comment:
What happens if user configures `enable_activation_checkpointing` and ac_mode is not None? Silently no use of AC/